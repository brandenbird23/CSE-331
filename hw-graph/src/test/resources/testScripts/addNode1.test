# addNode1 Test File
# tests if node can be added to a graph

# Create graph
CreateGraph graph1

# Create a node
AddNode graph1 node1

# Print the nodes in graph
ListNodes graph1