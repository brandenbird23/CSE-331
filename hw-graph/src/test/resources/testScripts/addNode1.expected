# addNode1 Test File
# tests if node can be added to a graph

# Create graph
created graph graph1

# Create a node
added node node1 to graph1

# Print the nodes in graph
graph1 contains: node1